tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -175,6 +179,23 @@ def client(self, old_report, new_report):
             self.breaking_changes.append(msg)
         return
 
+    def compare_operation_function(self):
+        '''
+        Record changelog like rename operation.delete to operation.begin_delete change
+        instead of using remove operation.delete or add operation.begin_delete
+        '''
+        while self.remove_operations:
+            op, old_function  = self.remove_operations.pop().split('.')
+            if f'{op}.begin_{old_function}' in self.add_operations:
+                self.add_operations.remove(f'{op}.begin_{old_function}')
+                self.breaking_changes.append(_RENAME_OPERATION.format(op, old_function, op, f'begin_{old_function}'))
+            else:
+                self.breaking_changes.append(_REMOVE_OPERATION.format(op, old_function))
```

Comment:
`f'begin_{old_function}'` is used for 3 times. It is better to add `new_function = f'begin_{old_function}'`